iset.mm

```diff
@@ -150553,6 +150553,277 @@ the additive structure must be abelian (see ~ ringcom ), care must be
+      AMZGAMVMVOQVSVTVEVGWBVFACEVJGIVRHUJTVHACEGVPVQVHVECURMVSCUKULWAUMACDFVKGH
+      KUNUOVHVOFCUPPZDOZFVHVNWCFDVEVGVNWCQVFCDEWCVJGIVRKWCSZUSTUTVEVFWDFQVGACDW
+      CFHKWEVAVBVCVD $.
+
+    $( A cancellation law for division.  ( ~ divcanap3 analog.)  (Contributed
+       by Mario Carneiro, 2-Jul-2014.)  (Revised by Mario Carneiro,
+       18-Jun-2015.) $)
+    dvrcan3 $p |- ( ( R e. Ring /\ X e. B /\ Y e. U ) ->
+      ( ( X .x. Y ) ./ Y ) = X ) $=
+      ( crg wcel w3a co cur cfv wceq simp1 a1i simp2 cbs cui csrg ringsrg simp3
+      unitcld dvrass syl13anc eqid dvrid 3adant2 oveq2d ringridm 3adant3 3eqtrd
+      syl ) CLMZFAMZGEMZNZFGDOGBOZFGGBOZDOZFCPQZDOZFVAURUSGAMUTVBVDRURUSUTSZURU
+      SUTUAVAACEGACUBQRVAHTECUCQRVAITVAURCUDMVGCUEUQURUSUTUFZUGVHABCDEFGGHIJKUH
+      UIVAVCVEFDURUTVCVERUSBCEVEGIJVEUJZUKULUMURUSVFFRUTACDVEFHKVIUNUOUP $.
+  $}
+
+  ${
+    dvreq1.b $e |- B = ( Base ` R ) $.
+    dvreq1.o $e |- U = ( Unit ` R ) $.
+    dvreq1.d $e |- ./ = ( /r ` R ) $.
+    dvreq1.t $e |- .1. = ( 1r ` R ) $.
+    $( A cancellation law for division.  ( ~ diveqap1 analog.)  (Contributed by
+       Mario Carneiro, 28-Apr-2016.) $)
+    dvreq1 $p |- ( ( R e. Ring /\ X e. B /\ Y e. U ) ->
+                   ( ( X ./ Y ) = .1. <-> X = Y ) ) $=
+      ( crg wcel w3a co wceq cfv oveq1 a1i 3adant2 eqid dvrcan1 wa cbs cui csrg
+      cmulr ringsrg adantr simpr unitcld ringlidm syldan eqeq12d imbitrid dvrid
+      eqeq1d syl5ibrcom impbid ) CLMZFAMZGDMZNZFGBOZEPZFGPZVEVDGCUGQZOZEGVGOZPV
+      CVFVDEGVGRVCVHFVIGABCVGDFGHIJVGUAZUBUTVBVIGPZVAUTVBGAMVKUTVBUCZACDGACUDQP
+      VLHSDCUEQPVLISUTCUFMVBCUHUIUTVBUJUKACVGEGHVJKULUMTUNUOVCVEVFGGBOZEPZUTVBV
+      NVABCDEGIJKUPTVFVDVMEFGGBRUQURUS $.
+  $}
+
+  ${
+    ringinvdv.b $e |- B = ( Base ` R ) $.
+    ringinvdv.u $e |- U = ( Unit ` R ) $.
+    ringinvdv.d $e |- ./ = ( /r ` R ) $.
+    ringinvdv.o $e |- .1. = ( 1r ` R ) $.
+    ringinvdv.i $e |- I = ( invr ` R ) $.
+    $( Write the inverse function in terms of division.  (Contributed by Mario
+       Carneiro, 2-Jul-2014.) $)
+    ringinvdv $p |- ( ( R e. Ring /\ X e. U ) -> ( I ` X ) = ( .1. ./ X ) ) $=
+      ( crg wcel wa co cfv cmulr wceq a1i eqid cinvr cdvr simpl ringidcl adantr
+      cbs cui simpr dvrvald ringinvcl ringlidm syldan eqtr2d ) CMNZGDNZOZEGBPEG
+      FQZCRQZPZURUQABCUSDFEGACUGQSUQHTUSUSSUQUSUAZTDCUHQSUQITFCUBQSUQLTBCUCQSUQ
+      JTUOUPUDUOEANUPACEHKUEUFUOUPUIUJUOUPURANUTURSACDFGILHUKACUSEURHVAKULUMUN
+      $.
+  $}
+
+  ${
+    $d x y B $.  $d x y z K $.  $d x y z L $.  $d x y z ph $.
+    rngidpropd.1 $e |- ( ph -> B = ( Base ` K ) ) $.
+    rngidpropd.2 $e |- ( ph -> B = ( Base ` L ) ) $.
+    rngidpropd.3 $e |- ( ( ph /\ ( x e. B /\ y e. B ) ) ->
+      ( x ( .r ` K ) y ) = ( x ( .r ` L ) y ) ) $.
+    rngidpropdg.k $e |- ( ph -> K e. V ) $.
+    rngidpropdg.l $e |- ( ph -> L e. W ) $.
+    $( The ring identity depends only on the ring's base set and multiplication
```

Comment:
"ring identity" => "ring unity"

Reply:
Looks like we missed this one in the recent renames.  Renamed in set.mm and iset.mm.

Reply:
I looked for certain patterns only for the renaming (mainly "unity", "neutral"), therefore it is still possible that I missed some more...